lib/libc/string/memset.c

```diff
@@ -58,7 +58,7 @@ bzero(void *dst0, size_t length)
 #include <string.h>
 
 #define	RETURN	return (dst0)
-#define	VAL	c0
+#define	VAL	(u_char)c0
```

Comment:
This seems out of place now. Can you split this out of the commit it is in and say it's an explicit cast because you are assigning through a u_char pointer?